server/fleet/queries.go

```diff
@@ -12,6 +12,12 @@ import (
 	"github.com/ghodss/yaml"
 )
 
+const (
+	AggregatedStatsTypeScheduledQuery = "scheduled_query"
+)
+
+type AggregatedStatsType string
```

Comment:
feels a bit odd to have this type inside of `queries` since there are non-query related aggregatedStats still in `aggregated_stats.go`.  Maybe this should live in a new `server/fleet/aggregated_stats.go`?

A bit out of scope, but it's worth considering using `iota` to convert aggregatedStats constants to enums

Reply:
Created `aggregated_stats.go`